aspnet/mvc/controllers/areas.rst

```diff
@@ -60,11 +78,17 @@ Once you've defined the folder hierarchy, you need to tell MVC that each control
       [Area("Products")]
       public class HomeController : Controller
       {
-          // GET: /<controller>/
+          // GET: /Products/Home/Index
```

Comment:
> The home controller is invoked like a products controller? Is that right?

I am not clear...This is HomeController in Products area and the path to reach it would be either `/products/home/index` (right if the defaults are considered this is not always true)